src/display/text_layer.js

```diff
@@ -585,7 +594,7 @@ var renderTextLayer = (function renderTextLayerClosure() {
         for (i = 0, ii = this._textDivs.length; i < ii; i++) {
           div = this._textDivs[i];
           div.style.padding = 0;
-          transform = div.dataset.originalTransform || '';
+          transform = divProperties.originalTransform || '';
```

Comment:
Fixed in the new commit. The structure was a bit strange here since the `for` loop and `div` fetching is equal in both the `if` and `else` branches, so I've slightly refactored that to prevent this issue.